tmt-app/src/Components/Home.js

```diff
@@ -94,26 +103,42 @@ function Home() {
             <Typography variant="CustomHeading2">Take the Quiz</Typography>
           </PrimaryBtn>
 
-          <div className="Home_Button_Main_Link">
-            <Link
-              sx={{ color: "Boba.main" }}
-              underline="none"
-              target="_blank"
-              rel="noopener noreferrer"
-              aria-label="Learn more about other different majors"
-              component={RouterLink}
-              to="/guide"
-            >
+          <FlexContainer sx={{ marginTop: "4.5rem", flexDirection: "column" }}>
+            <Button onClick={toggleDisplayGuide}>
               <Typography variant="CustomSubHeading">
                 See all Majors/Careers
               </Typography>
-            </Link>
-            <ArrowRightAltIcon
-              sx={{ paddingTop: "3px", paddingLeft: "10px" }}
-            />
-          </div>
+              {displayGuide ? (
+                <ArrowUpwardIcon
+                  sx={{ paddingTop: "3px", paddingLeft: "10px" }}
+                />
+              ) : (
+                <ArrowDownwardIcon
+                  sx={{ paddingTop: "3px", paddingLeft: "10px" }}
+                />
+              )}
+            </Button>
+          </FlexContainer>
         </FlexContainer>
       </FlexContainer>
+      <Container
+        sx={{
+          display: "flex",
+          alignItems: "flex-start",
+          paddingTop: "8rem",
+          opacity: displayGuide ? "100%" : "0%",
+          visibility: displayGuide ? "visible" : "hidden",
+          transition: "450ms ease-in-out",
+          position: "absolute",
+          left: "50%",
+          transform: "translate(-50%, 0)",
+          bottom: displayGuide ? "-50%" : "-10%",
+          zIndex: "-10",
```

Comment:
How are you determining what zIndex value to use? Are any of the MUI provided zIndex values preferable? https://mui.com/material-ui/customization/z-index/